src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageAnalyzerResultProvider.cs

```diff
@@ -115,8 +120,10 @@ void GetResults<T>(string name, string action, IReadOnlyCollection<Operation<T>>
                             {
                                 results.UnionWith(s.OperationDetails.Details.Select(s => new AnalyzeResult()
                                 {
-                                    RuleId = Id,
-                                    RuleName = this.Name,
+                                    RuleId = ruleId,
+                                    RuleName = ruleName,
+                                    FullDescription = FullDescription,
```

Comment:
Oh the FullDescription should also be passed to the method similar to ruleId and ruleName. For now the helpUrl can just be the common one.

Reply:
what do u suggest for fulldescription?

Reply:
"References that need to be deleted in order to upgrade the project to chosen TFM" similarly for the other areas accordingly "Packages" or "Framework References" and "added" or "deleted" .

Reply:
changed!